Dockerfile

```diff
@@ -16,9 +15,7 @@ RUN apt update && DEBIAN_FRONTEND=noninteractive apt install ros-noetic-desktop-
 RUN echo "source /opt/ros/noetic/setup.bash" >> /root/.bashrc
 RUN echo "source /usr/share/gazebo/setup.sh" >> /root/.bashrc
 RUN source /root/.bashrc
-RUN apt update && apt install python3-rosdep python3-rosinstall python3-rosinstall-generator python3-wstool python3-serial build-essential -y
-RUN rosdep init
-RUN rosdep update
+RUN apt update && apt install python3-rosinstall-generator python3-wstool python3-serial  -y
```

Comment:
This was necessary?

Reply:
These one's already come installed in the ROS image. 

I think we can also remove gnupg on line 8 since the ROS image has gnupg2.

Reply:
https://hub.docker.com/layers/library/ros/noetic-ros-base-focal/images/sha256-2bb6821186d86445b5f0b2da7a4ccc36dbc26707ae18b8c8b52184baf0c5fd11?context=explore

Reply:
Yeah, if possible, remove everything that already comes with the ROS image.

Reply:
Let's try to have a minimal Dockerfile, using most of the default ROS image when possible.